frontend/src/components/Timeline.jsx

```diff
@@ -1,7 +1,8 @@
 import { Box, Button, Card, CardContent, Typography , CardActionArea } from "@mui/material";
 
 const formatDateTime = (dateTimeString) => {
-  const options = {timeZone: "Asia/Tokyo", year: "numeric", month: "2-digit", day: "2-digit", hour: "2-digit", minute: "2-digit" };
+  const getTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  const options = {timeZone: getTimeZone, year: "numeric", month: "2-digit", day: "2-digit", hour: "2-digit", minute: "2-digit" };
```

Comment:
getTimeZone → timeZone にすることにより省略できるんだけど、これどうするべきなんだろ @sor4chi 

```suggestion
  const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
  const options = {timeZone, year: "numeric", month: "2-digit", day: "2-digit", hour: "2-digit", minute: "2-digit" };
```